account/accounts.go

```diff
@@ -16,27 +16,19 @@ import (
 
 	"github.com/status-im/extkeys"
 
-	gethkeystore "github.com/ethereum/go-ethereum/accounts/keystore"
-	gethcommon "github.com/ethereum/go-ethereum/common"
+	gethKeystore "github.com/ethereum/go-ethereum/accounts/keystore"
+	gethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/account/common"
+	accountErrors "github.com/status-im/status-go/account/errors"
 	"github.com/status-im/status-go/account/generator"
-	gocommon "github.com/status-im/status-go/common"
+	"github.com/status-im/status-go/account/types"
+	goCommon "github.com/status-im/status-go/common"
 	"github.com/status-im/status-go/eth-node/crypto"
 	"github.com/status-im/status-go/eth-node/keystore"
-	"github.com/status-im/status-go/eth-node/types"
+	ethTypes "github.com/status-im/status-go/eth-node/types"
```

Comment:
golang prefers `flatcase` for package names, not `snake_case` or `camelCase`
https://go.dev/blog/package-names#package-names

(as much as I'd dislike it, consistency is more important)